### PR TITLE
Add CI for openshift/grafana-tempo-operator

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

**Comment by pavolloffay** [comment]
@IshwarKanse could you please review?

File: ci-operator/config/openshift/grafana-tempo-operator/openshift-grafana-tempo-operator-main.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+binary_build_commands: go install ./
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.20
```
Comment (IshwarKanse) [inline ci-operator/config/openshift/grafana-tempo-operator/openshift-grafana-tempo-operator-main.yaml:6]: Looks like golang-1.20 image stream is not yet available. There is a [issue](https://redhat-internal.slack.com/archives/CBN38N3MW/p1687379872277429) reported for the same in the forum-testplatform channel.

Comment (pavolloffay) [reply ci-operator/config/openshift/grafana-tempo-operator/openshift-grafana-tempo-operator-main.yaml]: I have switched it to 1.19

**Pull request merged**